Double-click-to-edit/DoubleClickToEdit.plugin.js

```diff
@@ -100,9 +102,12 @@ module.exports = class DoubleClickToEdit {
 		//Pretty neat.
 		return () => {
 			const [replyState, setReplyState] = React.useState(this.doubleClickToReplySetting),
+				[replyActionKeyState, setReplyActionKeyState] = React.useState(this.replyActionKey),
+				[editActionKeyState, setEditActionKeyState] = React.useState(this.editActionKey),
 				[copyState, setCopyState] = React.useState(this.copyBeforeAction),
 				[copyModifierState, setCopyModifierState] = React.useState(this.copyBeforeActionModifier);
```

Comment:
```suggestion
			//Edit
			const [editEnableModifier, setEditEnableModifier] = React.useState(this.doubleClickToEditModifier),
				[editModifier, setEditModifier] = React.useState(this.editModifier),
				//Reply
				[reply, setReply] = React.useState(this.doubleClickToReply),
				[replyEnableModifier, setReplyEnableModifier] = React.useState(this.doubleClickToReplyModifier),
				[replyModifier, setReplyModifier] = React.useState(this.replyModifier),
				//Copy
				[copy, setCopy] = React.useState(this.doubleClickToCopy),
				[copyModifier, setCopyModifier] = React.useState(this.copyModifier);
```
I've revamped the entire settings screen, starting with the way the settings are build